interfaces/admin_cli.py

```diff
@@ -0,0 +1,501 @@
+    It will be launched when the admin choose #3 and enter do_manage_volunteer_account().
+    """
+    admin_options = {"logout": 0, "create_volunteer": 1, "view_volunteer": 2,
+                     "edit_volunteer": 3, "deactivate_volunteer": 4, "reactivate_volunteer": 5,
+                     "delete_volunteer": 6,
+                     "return_main_menu": "r"}
+
+    def volunteer_menu(self) -> None:
+        print("\n\033[100m\033[4m\033[1m{}\033[0m\n".format("Manage Volunteer Accounts"))
+        print(
+            "[ 1 ] Create a new volunteer \n"
+            "[ 2 ] View a volunteer profile\n"
+            "[ 3 ] Edit a volunteer profile\n"
+            "[ 4 ] De-activate a volunteer Account\n"
+            "[ 5 ] Re-activate a volunteer Account\n"
+            "[ 6 ] Delete a volunteer Account\n\n"
+            "[ R ] Return to Main Menu\n"
+            "[ 0 ] Log-out\n")
+
+    def preloop(self) -> None:
+        """
+        Display manage volunteer menu.
+        """
+        self.volunteer_menu()
+
+    def return_previous_page(self) -> bool:
+        while True:
+            option = input("Enter R to return back to main menu. > ")
+            if option.isalpha() and option.upper() == 'R':
+                ManageVolunteerMenu(self.user).cmdloop()
+                break
+            else:
+                print("\033[31m {}\033[00m".format("** Invalid input. Enter R to return back to main menu."))
+                continue
+
+    # ----- basic commands for volunteer account management -----
+    def do_create_volunteer(self, arg):
+        """
+        #1 Create a volunteer
+        """
+        print("\033[100m\033[4m\033[1m{}\033[0m ".format("Create a new volunteer account"))
+        # STEP 1: validate if plan exists
+        while True:
+            plan = input("Enter the plan that the new volunteer belongs to:")
+            try:
+                find_plan = plan_controller.find_plan(plan)
+                plan = find_plan
+                break
+            except ControllerError:
+                print(f"\033[31m * Plan {plan} not found. Please re-enter plan name. \033[00m")
+                continue
+
+        # STEP 2: validate if camp exists
+        while True:
+            camp = input("Enter the camp that the new volunteer belongs to:")
+            try:
+                find_camp = plan_controller.find_camp(plan=plan, camp_name=camp)
+                camp = find_camp
+                break
+            except ControllerError:
+                print(f"\033[31m * Camp {camp} not found. Please re-enter camp name. \033[00m")
+                continue
+
+        # STEP 3: other user inputs
+        username = input("Enter volunteer's username:")
+        password = input("Enter volunteer's password:")
+        firstname = input("Enter volunteer's first name:")
+        lastname = input("Enter volunteer's last name:")
+        phone = input("Enter volunteer's phone (start from + sign and national code):")
```

Comment:
Again, for these inputs above, if you have a chance could you please add a space between the colon and "? Very minor readability thing, so if you don't have time, no worries!